canonical-websites/vanillaframework.io
# Update ./run and remove gulp

Update the build process to use the latest ./run script. Remove gulp to
favour the ./run script pocess.

## Done

- Updated run script
- Copied over configuration from canonical-websites/design.ubuntu.com
- Removed gulp

## QA

- `./run clean`
- `./run`
- View http://0.0.0.0:8014

Everything should run well and look like the currently live site
